modules/cnv-dedicated-resources.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-dedicated-resources-vm-template"]
+:cnv-vm-template:
+:object: virtual machine template
+:object-gui: Virtual Machine Template
+endif::[]
+
+ifeval::["{context}" == "cnv-dedicated-resources-vm"]
+:cnv-vm:
+:object: virtual machine
+:object-gui: Virtual Machine
+endif::[]
+
+[id="cnv-dedicated-resources_{context}"]
+
+= Enabling dedicated resources for a {object}
+
+ifdef::cnv-vm-template[]
+Use this procedure to enable dedicated resources and schedule workloads for virtual
+machine templates.
+endif::cnv-vm-template[]
+ifdef::cnv-vm[]
+Use this procedure to enable dedicated resources and schedule workloads for virtual
+machines.
+endif::cnv-vm[]
+
+.Procedure
+
+To enable dedicated resources, do the following:
+
+. Select *Workloads* > *{object-gui}*.
```

Comment:
Some markup issues here. Suggest:

. Click \*Workloads\* -> \*{object-gui}s\* from the side menu.
. Select a {object} to open the \*{object-gui} Overview\* page.

Reply:
Done. See next section for complete changes.